usr/lib/pkcs11/ep11_stdll/ep11_specific.c

```diff
@@ -1723,6 +1731,8 @@ static CK_RV ep11_resolve_lib_sym(void *hdl)
     *(void **)(&dll_xcpa_queryblock) = dlsym(hdl, "xcpa_queryblock");
     *(void **)(&dll_xcpa_internal_rv) = dlsym(hdl, "xcpa_internal_rv");
 
+    *(void **)(&dll_m_get_xcp_info) = dlsym(hdl, "m_get_xcp_info");
```

Comment:
Is the m_get_xcp_info symbol exported by all ep11 lib versions? dll_m_get_xcp_info is dereferenced later w/o NULL checking.

Reply:
Good point. I added a NULL check. It should be available on all recent versions, but I don't know exactly when it was added.